src/renderer/apps/multisend/disbursements-state.ts

```diff
@@ -0,0 +1,127 @@
+import * as React from 'react'
+import electron from 'electron'
+import path from 'path'
+
+import useLocalStorageState from '../../state/localstorage-state'
+import DisbursementsDB, { Disbursement, Transfer } from './disbursementsdb'
+
+let _db: DisbursementsDB
+
+const disbursementsDB = (): DisbursementsDB => {
```

Comment:
While other comments should be easy to fix, this one might require larger changes. 

Overall, individual apps aren't really supposed to hold any persistent state, especially as a database stored in a local file. Apps are supposed to mainly depend on just blockchain for storing all the truly persistent state and can also use LocalStorage through `useLocalStorageState` helpers for storing app preferences or caching some small amount of state. 

Using a real database like this for individual apps is going to be problematic since it will add too much complexity for managing on disk state, for managing state cleanup and other stuff like that. This will also make it harder to more easily sandbox each individual app in the future, since each individual app shouldn't really be accessing local file system at all in any way.

I don't know how critical this database is for the app to function, but this will require rethinking to have it committed as part of the main Celo Terminal app.

Reply:
So, I just fixed the rest of the comments. 

This point is a bit critical for us, because we are using this database to keep track of the list of disbursements. We are expecting to have a big amount of disbursements, that's why the `useLocalStorageState` could be a trouble.
I don't know if you think that could be another approach that we can follow to persist some information.